aiidalab_widgets_base/aiida_visualizers.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import print_function
+import os
+
+import ipywidgets as ipw
+
+class ParameterDataVisualizer(ipw.HTML):
+    """Visualizer class for ParameterData object"""
+    def __init__(self, parameter, downloadable=True, **kwargs):
+        super(ParameterDataVisualizer, self).__init__(**kwargs)
+        import pandas as pd
+        self.value = '''
```

Comment:
Could you add a comment perhaps with a link here, explaining where the CSS is coming from?

Reply:
Sure.

Of course "the" link does not exist. I was basically acting on the standard html table settings.

I added a couple of lines describing what pandas exported object is, and how to control its appearance through css styling.